src/plugins/degenIntel/payment-config.ts

```
@@ -0,0 +1,389 @@
+
+    // Construct from server settings
+    const protocol = process.env.HTTPS === 'true' ? 'https' : 'http';
+    const host = process.env.SERVER_HOST || 'localhost';
+    const port = process.env.SERVER_PORT || '3000';
+
+    // Don't include port 80 for http or port 443 for https
+    if ((protocol === 'http' && port === '80') || (protocol === 'https' && port === '443')) {
+        return `${protocol}://${host}`;
+    }
+
+    return `${protocol}://${host}:${port}`;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because either this statement ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)